driver/ppm_events_public.h

```diff
@@ -798,6 +798,16 @@ or GPL2.txt for full copies of the license.
 #define PPM_MODULE_INIT_IGNORE_VERMAGIC     2
 #define PPM_MODULE_INIT_COMPRESSED_FILE     4
 
+/*
+ * bpf_commands 
+*/
+#define PPM_BPF_MAP_CREATE 		 (0<<1)
```

Comment:
Kernel API `bpf_cmd` is an enum, i'd follow the same values here (so 0,1,2,...)

Reply:
Also, i'd cover all existing values: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h#L866